### ISSUE TITLE
[Feature] I have a few questions regarding the InternVL2-8B-MPO model.

### ISSUE THREAD

**Issue by ajskdlf64** [OpenGVLab/InternVL#717]
### Motivation

1. Regarding Table 3, does the InternVL2-8B-SFT model utilize MMPR data with the chosen responses for supervised fine-tuning (SFT)? I am curious about the specific impact of the MMPR dataset.

2. Regarding Table 8, how does MathVista perform under the baseline condition when applying DPO? The results with DPO+ are quite competitive with MPO, so I would like to understand the standalone effectiveness of DPO.

3. Regarding Table 8, the MathVista score under DPO+ is reported as 66.4. Is this the CoT score? If so, could you also share the Direct score for comparison?

4. Regarding Figure 2, the paper mentions that the prompts used to create the MMPR dataset are publicly available, but I couldn't locate them. Could you kindly provide clarification or guidance on where to find them?

Thank you very much for your time and for the valuable contributions your work has made to the community. I look forward to hearing from you

**Comment by star580** [comment]
Hi~想问一下InternVL2-8B MPO文章里面提到_**We observed that when MLLMs are trained on large-scale preference datasets using direct preference optimization (DPO), they might fail to generate reasonable rationales and produce gibberish.**_ 很好奇直接进行DPO性能怎么样？有观察在普通DPO优化过程中chosen reward和reject reward的变化情况吗？

**Comment by Weiyun1025** [comment]
Sorry for the late response.

## Response to Q1
Yes, we utilize the chosen responses in MMPR for SFT. The purpose of such settings is to validate the effectiveness of Preference Optimization (PO), as PO includes additional supervision signals on the rejected responses.

## Response to Q2
In our early experiments, we observed that when the training data is large-scale, the resulting model struggle to follow the instruction and issues like repetition (e.g., the model failing to stop when the response should be complete) become more fluent. We attribute this phenomenon to the fact that DPO Loss is a discriminative loss, leading to the performance decline on the generation abilities. Therefore, we introduce the generation loss to maintain the generation ability

## Response to Q3
Yes, reported score of 66.4 is the CoT score. The direct score of MathVista under DPO+ is 65.8.

## Response to Q4
We collect most prompts from open-source datasets, as listed in Table 1. You can extract these prompts from the annotations of MMPR by disregarding the chosen and rejected responses.

Please let me know if you still have any further questions. I shall respond to you as soon as possible. Apologies once again for the delay in my reply.
